app/src/main/java/com/realityexpander/tasky/agenda_feature/domain/AgendaItem.kt

```diff
@@ -0,0 +1,32 @@
+package com.realityexpander.tasky.agenda_feature.domain
+
+import java.time.LocalDateTime
+
+// NOTE: Skeleton data structures for the Agenda feature
+// **temporary** for UI development
+
+abstract class AgendaItem(
+    val id: String,
+    val title: String,
+    val description: String,
+    val remindAt: LocalDateTime,
+) {
+    abstract fun isEmpty(): Boolean
```

Comment:
What does it mean for an AgendaItem to be empty? 🤔

Reply:
I will make them nullable instead of empty.